package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "react-capstone-covid",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@reduxjs/toolkit": "^1.5.1",
+    "@testing-library/jest-dom": "^4.2.4",
+    "@testing-library/user-event": "^7.1.2",
+    "react": "^17.0.2",
+    "react-bootstrap": "^2.0.0-beta.6",
+    "react-dom": "^17.0.2",
+    "react-icons": "^4.2.0",
+    "react-redux": "^7.2.3",
+    "react-router-dom": "^5.2.0",
+    "react-scripts": "4.0.3",
+    "redux-logger": "^3.0.6",
+    "redux-thunk": "^2.3.0"
+  },
```

Comment:
- [ ] You are required to use Redux in this project. Kindly install and implement the project with redux as per the project requirements.

Reply:
I am using redux. Please check my other comment at the end of this conversation!